### DISCUSSION TITLE
.gitignore template for Android: doesn't contain `.DS_Store`

### DISCUSSION THREAD

**Discussion by raphael-dario** [community/community#41024] [Repositories]
### Select Topic Area

Product Feedback

### Body

Hi everyone.
I think it would be cool if you could add the following in the `.gitignore` template for new Android repositories:
```
# mac
.DS_Store
```
It would certainly save a huge amount of space on Android repositories with commits pushed by people working on Mac.
Just a thought.

**Comment by r-cohen** [comment]
never mind, I just noticed there is an existing repository for gitignore templates:
https://github.com/github/gitignore
Will propose a PR there.

**Comment by ossanna16** [comment]
Hi there @raphael-dario and welcome to our community! Thank you for the feedback!
